File: app/src/main/java/com/example/calculator/activities/AboutActivity.java
package com.example.calculator.activities;

import android.os.Bundle;
import android.widget.ImageView;

import androidx.appcompat.app.AppCompatActivity;

import com.bumptech.glide.Glide;
import com.bumptech.glide.request.RequestOptions;
import com.example.calculator.R;

public class AboutActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_about);
        getSupportActionBar().setTitle("About");
        ImageView imageView = findViewById(R.id.nameImage);
        Glide.with(this)
                .load(R.drawable.name)
                .apply(RequestOptions.circleCropTransform())
                .into(imageView);
    }
}

File: app/src/main/java/com/inator/calculator/activities/AboutActivity.kt
package com.inator.calculator.activities

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.bumptech.glide.Glide
import com.bumptech.glide.request.RequestOptions
import com.inator.calculator.R
import kotlinx.android.synthetic.main.activity_about.*

class AboutActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_about)
        supportActionBar?.title = "About"
        Glide.with(this)
            .load(R.drawable.name)
            .apply(RequestOptions.circleCropTransform())
            .into(nameImage)
    }
}